manim/mobject/coordinate_systems.py

```diff
@@ -665,16 +746,32 @@ def get_lines_parallel_to_axis(
                     lines2.add(new_line)
         return lines1, lines2
 
-    def get_center_point(self):
+    def get_center_point(self) -> np.ndarray:
+        """Gets the center point from coordinates.
+
+        Returns
+        -------
+        np.ndarray
+            Returns the center point.
+        """
         return self.coords_to_point(0, 0)
 
     def get_x_unit_size(self):
+        # not yet implemented ?
```

Comment:
Why add this here? The implementation is here.

Reply:
The implementation is there but when we run the function it will raise a NotImplementedError

Reply:
Why though? It works perfectly fine...